Xext/xvdisp.c

```diff
@@ -624,6 +730,26 @@ XineramaXvPutImage(ClientPtr client);
 static int
 ProcXvPutImage(ClientPtr client)
 {
+    REQUEST(xvPutImageReq);
+    REQUEST_AT_LEAST_SIZE(xvPutImageReq);
```

Comment:
Should we remove this line from the `SingleXvPutImage` function above? Just like in other similar cases.

Reply:
indeed.